app/assets/javascripts/components/story/ExpandedStory/index.js

```diff
@@ -19,12 +19,22 @@ export class ExpandedStory extends React.Component {
     this.titleRef.current.focus();
   }
 
+  handleSaveStory = (story, project, from) => {
+    if (Story.needConfirmation(story._editing) && !window.confirm(
```

Comment:
What do you think about moving it to the action? That way we can make a reusable function there and then refactor this https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/components/story/CollapsedStory/CollapsedStoryStateActions.js#L34-L49 as well so we have this confirmation logic in a single place :smile: